test_rclcpp/CMakeLists.txt

```diff
@@ -338,6 +343,22 @@ if(BUILD_TESTING)
       TIMEOUT 30
       ${SKIP_TEST})
 
+        # Test that a user-defined signal handler is called on interrupt:
+    # after rclcpp::init has been called, but before rclcpp::shutdown has been called.
```

Comment:
@ivanpauno nit: 
```suggestion
    # Test that a user-defined signal handler is called on interrupt:
    # after rclcpp::init has been called, but before rclcpp::shutdown has been called.
```

Reply:
Fixed in https://github.com/ros2/system_tests/pull/485/commits/e259da3875ed7e1184a5f82315b95b627dd8c81d